scripts/test/unit-with-coverage

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+
+for pkg in $(go list ./... | grep -v /vendor/); do
+    ./scripts/test/unit \
+        -cover \
+        -coverprofile=profile.out \
+        -covermode=atomic \
+        ${pkg}
+    
+    if test -f profile.out; then
```

Comment:
The file is given as one argument, then copied to the other. Why not give the original file to the command and have it write directly? Are you trying to append it to a global coverage file?